osu.Game.Rulesets.Taiko/Objects/HitType.cs

```diff
@@ -18,6 +18,16 @@ public enum HitType
         /// <summary>
         /// A <see cref="Hit"/> that can be hit by the rim portion of the drum.
         /// </summary>
-        Rim
+        Rim,
+
+        /// <summary>
+        /// A <see cref="Hit"/> that can be hit by the centre portion of the drum.
+        /// </summary>
+        StrongCentre,
+
+        /// <summary>
+        /// A <see cref="Hit"/> that can be hit by the rim portion of the drum.
+        /// </summary>
+        StrongRim
```

Comment:
Hmm, I guess I didn't fully understand the implications of adding these.

I only really used it to affect sample playback in `DrumSamplePlayer`/`DrumSampleTriggerSource`, so can probably be done in a different way in order to avoid adding new `HitType`s.

Reply:
I think this is a much-of-a-muchness at this point, until we start to solidify how skinning should work with these new skins. It doesn't change behaviour because there are no internal usages that rely on the initial `Sample?.Get()` lookup path.